index.html

```diff
@@ -1,124 +1,996 @@
+    }
+    table tr {
+        padding: 0.5em 0
+    }
+    table tr,
+    table th,
+    table td {
+        display: block;
+        margin: 0 auto;
+        max-width: 21em;
+        text-align: center
+    }
+    table .number-data {
+        width: auto;
+        text-align: center
+    }
+    #forkme {
+        width: 100px;
+        height: 100px
+    }
+    #information .row .col-1 {
+        width: 100%;
+        padding: 0;
+        margin: 0
+    }
+    #information .row .col-2 {
+        width: 100%;
+        float: left
+    }
+    pre code#selectable {
+        width: 90%;
+        margin: 0 auto
+    }
+    [dir="rtl"] #information .row .col-2 {
+        float: right
+    }
+}
+
+.hll {
+    background-color: #ffc
+}
+
+.c {
+    color: #999988;
+    font-style: italic
+}
+
+.err {
+    color: #a61717;
+    background-color: #e3d2d2
+}
+
+.k {
+    font-weight: bold
+}
+
+.o {
+    font-weight: bold
+}
+
+.cm {
+    color: #999988;
+    font-style: italic
+}
+
+.cp {
+    color: #999999;
+    font-weight: bold
+}
+
+.c1 {
+    color: #999988;
+    font-style: italic
+}
+
+.cs {
+    color: #999999;
+    font-weight: bold;
+    font-style: italic
+}
+
+.gd {
+    color: #000000;
+    background-color: #fdd
+}
+
+.ge {
+    font-style: italic
+}
+
+.gr {
+    color: #a00
+}
+
+.gh {
+    color: #999
+}
+
+.gi {
+    color: #000000;
+    background-color: #dfd
+}
+
+.go {
+    color: #888
+}
+
+.gp {
+    color: #555
+}
+
+.gs {
+    font-weight: bold
+}
+
+.gu {
+    color: #aaa
+}
+
+.gt {
+    color: #a00
+}
+
+.kc {
+    font-weight: bold
+}
+
+.kd {
+    font-weight: bold
+}
+
+.kn {
+    font-weight: bold
+}
+
+.kp {
+    font-weight: bold
+}
+
+.kr {
+    font-weight: bold
+}
+
+.kt {
+    color: #445588;
+    font-weight: bold
+}
+
+.m {
+    color: #099
+}
+
+.s {
+    color: #b84
+}
+
+.na {
+    color: teal
+}
+
+.nb {
+    color: #999
+}
+
+.nc {
+    color: #999999;
+    font-weight: bold
+}
+
+.no {
+    color: #666
+}
+
+.ni {
+    color: purple
+}
+
+.ne {
+    color: #990000;
+    font-weight: bold
+}
+
+.nf {
+    color: #999999;
+    font-weight: bold
+}
+
+.nn {
+    color: #555
+}
+
+.nt {
+    color: teal
+}
+
+.nv {
+    color: teal
+}
+
+.ow {
+    font-weight: bold
+}
+
+.w {
+    color: #bbb
+}
+
+.mf {
+    color: #099
+}
+
+.mh {
+    color: #099
+}
+
+.mi {
+    color: #099
+}
+
+.mo {
+    color: #099
+}
+
+.sb {
+    color: #b84
+}
+
+.sc {
+    color: #b84
+}
+
+.sd {
+    color: #b84
+}
+
+.s2 {
+    color: #9c7645
+}
+
+.se {
+    color: #b84
+}
+
+.sh {
+    color: #b84
+}
+
+.si {
+    color: #b84
+}
+
+.sx {
+    color: #b84
+}
+
+.sr {
+    color: olive
+}
+
+.s1 {
+    color: #9c7645
+}
+
+.ss {
+    color: #b84
+}
+
+.bp {
+    color: #999
+}
+
+.vc {
+    color: teal
+}
+
+.vg {
+    color: teal
+}
+
+.vi {
+    color: teal
+}
+
+.il {
+    color: #099
+}
+    .container {
+    margin:0 auto; /* this will center the page */
+    width:960px; /* use your width here */
+    }
+  </style>
```

Comment:
Can these styles be removed and added to `/css/styles.css` with a reference to the external stylesheet file instead?  Internal styles are considered bad practice.